src/queryPublisherTemplate.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The filename here should match other json resources and use snake case

Reply:
Same name convention as the other query template, at least